NFIQ2/NFIQ2Algorithm/src/tool/nfiq2_ui_refresh.cpp

```diff
@@ -859,18 +837,33 @@ main(int argc, char **argv)
 	double timeInit = 0.0;
 	timerInit.startTimer();
 
-	std::string modelFile;
-	std::string hash;
+	std::shared_ptr<NFIQ::ModelInfo> modelInfoObj {};
```

Comment:
Shouldn't need to be a `shared_ptr`. Other code changes should make this possible.